tests/perfalert/test_alert_modification.py

```diff
@@ -1,10 +1,35 @@
 import datetime
 
 import pytest
-from django.core.exceptions import ValidationError
 
+from django.core.exceptions import ValidationError
 from treeherder.perf.models import (PerformanceAlert,
                                     PerformanceAlertSummary)
+from treeherder.model.models import (MachinePlatform,
+                                     Option,
+                                     OptionCollection)
+from treeherder.perf.models import PerformanceSignature
```

Comment:
The ordering here is generating a travis linter error: https://travis-ci.org/mozilla/treeherder/jobs/161086026

You can find out how to run the linters yourself here: http://treeherder.readthedocs.io/common_tasks.html#running-the-tests